lenskit-eval/src/main/java/org/lenskit/eval/traintest/DataSet.java

```diff
@@ -286,9 +306,31 @@ public static DataSetBuilder copyBuilder(DataSet data) {
      * @return The data source.
      */
     private static DataSet loadDataSet(JsonNode json, URI base, String name, int part) throws IOException {
+        System.out.println(json);
```

Comment:
stray `println` - delete this. Any debugging output should go through the logger.